website/src/components/pageStateSelectors/CompareVariantsToBaselineStateSelector.tsx

```diff
@@ -0,0 +1,89 @@
+import { type compareToBaselineViewKey } from '../../views/viewKeys.ts';
+
+export function CompareVariantsToBaselineStateSelector({
+    locationFilterConfig,
+    dateRangeFilterConfig,
+    baselineFilterConfig,
+    variantFilterConfigs,
+    organismViewKey,
+    organismsConfig,
+    pageState,
+}: {
+    locationFilterConfig: LocationFilterConfig;
+    dateRangeFilterConfig: DateRangeFilterConfig;
+    baselineFilterConfig: VariantFilterConfig;
+    variantFilterConfigs: Map<Id, VariantFilterConfig>;
+    organismViewKey: OrganismViewKey & `${string}.${typeof compareToBaselineViewKey}`;
+    organismsConfig: OrganismsConfig;
+    pageState: CompareVariantsData;
+}) {
+    const [location, setLocation] = useState<LapisLocation>(locationFilterConfig.initialLocation);
+    const [dateRange, setDateRange] = useState<DateRangeOption>(dateRangeFilterConfig.initialDateRange);
+    const [baselineFilterConfigState, setBaselineFilterConfigState] =
+        useState<VariantFilterConfig>(baselineFilterConfig);
+
+    const [variantConfigs, setVariantConfigs] = useState<Map<Id, VariantFilterConfig>>(variantFilterConfigs);
+
+    const view = useMemo(() => new Routing(organismsConfig), [organismsConfig]).getOrganismView(organismViewKey);
+
+    const newPageState = useMemo(() => {
+        const variants = new Map(
+            Array.from(variantConfigs).map(([id, variantFilterConfig]) => {
+                return [id, toVariantFilter(variantFilterConfig)];
+            }),
+        );
+
+        return {
+            ...pageState,
+            datasetFilter: {
+                location,
+                dateRange,
+            },
+            variants,
+            baselineFilter: toVariantFilter(baselineFilterConfigState),
+        };
+    }, [variantConfigs, pageState, location, dateRange, baselineFilterConfigState]);
+
+    return (
+        <div className='flex flex-col gap-6 bg-gray-50 p-2'>
+            <div>
+                <SelectorHeadline>Filter dataset</SelectorHeadline>
+                <BaselineSelector
+                    onLocationChange={setLocation}
+                    locationFilterConfig={locationFilterConfig}
+                    onDateRangeChange={setDateRange}
+                    dateRangeFilterConfig={dateRangeFilterConfig}
+                />
+            </div>
+            <div>
+                <SelectorHeadline>Baseline Filter</SelectorHeadline>
+                <VariantSelector
+                    onVariantFilterChange={setBaselineFilterConfigState}
+                    variantFilterConfig={baselineFilterConfigState}
+                />
+            </div>
+            <div>
+                <SelectorHeadline>Variant Filters</SelectorHeadline>
+                <VariantsSelector
+                    variantFilterConfigs={variantConfigs}
+                    setVariantFilterConfigs={setVariantConfigs}
+                    emptyVariantFilterConfig={view.pageStateHandler.getEmptyVariantFilterConfig()}
```

Comment:
I think we can simply things a bit by making this a callback:
```suggestion
                    emptyVariantFilterConfigProvider={view.pageStateHandler.getEmptyVariantFilterConfig}
```
Then we can get rid of `copyVariantFilterConfig`, because the provider always gets us a new object already anway.